src/material/list/_list-theme.scss

```diff
@@ -180,5 +184,8 @@
 @mixin _theme-from-tokens($tokens) {
   @if ($tokens != ()) {
     @include mdc-list-theme.theme(map.get($tokens, tokens-mdc-list.$prefix));
```

Comment:
while you're here, you might as well change this to use `get-tokens-for` instead of `map.get`, that way you can remove the `@if` guard too